stdlib/public/core/CompilerProtocols.swift

```diff
@@ -395,6 +395,14 @@ public protocol ExpressibleByExtendedGraphemeClusterLiteral
   init(extendedGraphemeClusterLiteral value: ExtendedGraphemeClusterLiteralType)
 }
 
+extension ExpressibleByExtendedGraphemeClusterLiteral
+    where Self.ExtendedGraphemeClusterLiteralType == Character
```

Comment:
Drive-by nit: indents are two spaces in this file; otherwise a delightful addition!